### PR TITLE
Transition nextclade 2->3

### PR DESCRIPTION
**Pull request by corneliusroemer**
Update nextclade3 version

Add nextclade2 in build.sh for stability until bioconda update is sorted out

Fails currently as we don't have the 3.0.0 gh release ye

## Checklist

- [ ] Checks pass

### CODE REVIEW

File: src/build.sh [hunk 1]
```diff
@@ -6,5 +6,11 @@ mkdir -p "$PREFIX"/bin
 cp -v "$SRC_DIR"/envdir "$PREFIX"/bin/envdir
 
 # Nextclade v3
-curl -fsSL -o "$PREFIX"/bin/nextclade3 https://github.com/nextstrain/nextclade/releases/download/3.0.0-alpha.2/nextclade-$("$SRC_DIR"/target-triple)
```
Comment (victorlin) [inline src/build.sh:9]: I see the existing addition of `nextclade3` here as temporary. I would just wait for Bioconda to be updated so `nextclade` is v3, remove this `curl` command, and rebuild a new version of `nextstrain-base`.

**Review by victorlin — CHANGES_REQUESTED** [review]

File: src/build.sh (hunk 1 above)
Comment (corneliusroemer) [reply src/build.sh:9]: yes exactly, recipe is approved over on bioconda, just be on the index in <1hr

Comment (victorlin) [reply src/build.sh:9]: Linking the Bioconda update here: https://github.com/bioconda/bioconda-recipes/pull/45259

Comment (corneliusroemer) [reply src/build.sh:9]: should be there any minute as PR has been merged

**Comment by corneliusroemer** [comment]
I think we can now get rid of all the nextclade 2/3 downloading in `build.sh` altogether

**Pull request closed without merging**